homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
     ),
     "nomapnt": SensorEntityDescription(
         key="nomapnt",
-        name="UPS Nominal Apparent Power",
+        translation_key="nominal_apparent_power",
         native_unit_of_measurement=UnitOfApparentPower.VOLT_AMPERE,
         device_class=SensorDeviceClass.APPARENT_POWER,
     ),
     "numxfers": SensorEntityDescription(
         key="numxfers",
         translation_key="transfer_count",
-        name="UPS Transfer Count",
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
     "outcurnt": SensorEntityDescription(
         key="outcurnt",
-        name="UPS Output Current",
+        translation_key="output_current",
         native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "outputv": SensorEntityDescription(
         key="outputv",
-        name="UPS Output Voltage",
+        translation_key="output_voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "reg1": SensorEntityDescription(
         key="reg1",
         translation_key="register_1_fault",
-        name="UPS Register 1 Fault",
         entity_registry_enabled_default=False,
     ),
     "reg2": SensorEntityDescription(
         key="reg2",
         translation_key="register_2_fault",
-        name="UPS Register 2 Fault",
         entity_registry_enabled_default=False,
     ),
     "reg3": SensorEntityDescription(
         key="reg3",
         translation_key="register_3_fault",
-        name="UPS Register 3 Fault",
         entity_registry_enabled_default=False,
     ),
     "retpct": SensorEntityDescription(
         key="retpct",
         translation_key="restore_capacity",
-        name="UPS Restore Requirement",
         native_unit_of_measurement=PERCENTAGE,
     ),
     "selftest": SensorEntityDescription(
         key="selftest",
         translation_key="self_test_result",
-        name="UPS Self Test result",
     ),
     "sense": SensorEntityDescription(
         key="sense",
         translation_key="sensitivity",
-        name="UPS Sensitivity",
         entity_registry_enabled_default=False,
     ),
     "serialno": SensorEntityDescription(
         key="serialno",
         translation_key="serial_number",
-        name="UPS Serial Number",
         entity_registry_enabled_default=False,
     ),
     "starttime": SensorEntityDescription(
         key="starttime",
         translation_key="startup_time",
-        name="UPS Startup Time",
     ),
     "statflag": SensorEntityDescription(
         key="statflag",
         translation_key="online_status",
-        name="UPS Status Flag",
         entity_registry_enabled_default=False,
     ),
     "status": SensorEntityDescription(
         key="status",
         translation_key="status",
-        name="UPS Status",
     ),
```

Comment:
What does this return?

Reply:
`statflag` returns a hex value indicating the status of whether it's online, charging, or on battery etc. `status` is the English version of the status (e.g., "ONLINE"). 

We should probably deprecate `statflag` and rely on `status` instead since [it's available in all APC UPSes but `statflag` isn't](https://linux.die.net/man/8/apcaccess#:~:text=apcaccess%20is%20a%20program%20which,installed%2C%20properly%20configured%20and%20running.).